src/components/MAPI/organizations/__tests__/OrganizationList.tsx

```diff
@@ -272,14 +273,19 @@ describe('OrganizationIndex', () => {
     });
     fireEvent.click(cancelButton);
 
-    expect(screen.queryByText('not-created-org')).not.toBeInTheDocument();
-
     expect(
       screen.queryByRole('button', {
         name: 'Create organization',
       })
     ).not.toBeInTheDocument();
 
+    expect(screen.queryByText('not-created-org')).not.toBeInTheDocument();
+    await waitFor(() =>
+      expect(
+        screen.queryByDisplayValue('not-created-org')
+      ).not.toBeInTheDocument()
+    );
```

Comment:
Just for the future: there's `waitForElementToBeRemoved()`, which is made specifically for this use case